src/app/contexts/EventContext/useWindowEvent/index.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+
+export const useWindowEvent = (
+  event,
+  handler,
+  passive = false,
+  _window = null,
```

Comment:
Howcome passing window in? Is it for testing? We can mock/stub the window in the tests to avoid needing this :)

Reply:
Removed!